waltz-ng/client/common/components/multi-select-tree-control/multi-select-tree-control.js

```diff
@@ -47,10 +45,9 @@ const initialState = {
 };
 
 
-function expandSearchNotes(hierarchy = []) {
-    return hierarchy.length < 6  // pre-expand small trees
-        ? _.clone(hierarchy)
-        : [];
+function haltEvent() {
+    preventDefault(event);
```

Comment:
Don't we need to declare `event` as a function argument and pass it through?

Reply:
`event` is a global